pkg/mock/nsxop.ini

```diff
@@ -0,0 +1,45 @@
+[DEFAULT]
+
+debug = True
```

Comment:
Please remove options which are not used in current operator code

Reply:
done